templates/field--field-funding-contact-name.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="funding-ct--summary-section--contact-info-name">
+    {% for item in items %}
+        <span class="wdn-icon-user" aria-hidden="true"></span><span class="wdn-text-hidden">contact icon</span>
```

Comment:
This field only permits one value at this time. Changing expression to use index 0 instead.